src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,68 @@
+import { NotificationProps } from './types';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { Icon } from '../Icon';
+import { iconTypes } from '../Icon/collection';
+import { notificationStyles } from './Notification.styles';
+
+const NotificationStyled = styled.div`
+    ${notificationStyles.initialStyles}
+`;
+
+const BoxStyled = styled.div`
+    ${notificationStyles.box}
+`;
+
+const SpanStyled = styled.span`
+    ${notificationStyles.message}
+`;
+
+const ParagraphStyled = styled.p`
+    ${notificationStyles.title}
+`;
+
+const FlexStyled = styled.div`
+    ${notificationStyles.flex}
+`
+
+const Notification: React.FC<NotificationProps> = ({
+    id = String(Date.now()),
+    icon,
+    message = "Kresimir: Thank you for sharin..",
+    title = "New Message",
+    active = false,
+}: NotificationProps) => {
+    const [visible, setVisible] = useState(active);
+
+    useEffect(() => {
```

Comment:
i could be wrong but i dont think you need this useEffect as `active` is a property supplied to this component, if `active` was to change the component would re-redner anyway

Reply:
Unfortunately I used without useEffect but switching active props in storybook didnt close the notification somehow